src/cephadm/CMakeLists.txt

```diff
@@ -2,3 +2,16 @@ if(WITH_TESTS)
   include(AddCephTest)
   add_tox_test(cephadm TOX_ENVS py3 mypy flake8)
 endif()
+
+add_custom_command(
+    OUTPUT "${CMAKE_SOURCE_DIR}/src/cephadm/cephadm"
```

Comment:
presumably, we should redistribute the "compiled" cephadm which is built from the dist tarball. so i think we should 

1. create the temporary files and `cephadm` in `${CMAKE_BINARY_DIR}` not `${CMAKE_SOURCE_DIR}`
2. install the built `cephadm` instead. i.e. update `debian/rules` and `ceph.spec.in` accordingly. currently, we just copy `src/cephadm/cephadm` to `sbin/cephadm`. after the change, we should install it using `install(PROGRAMS ...)` in cmake.